platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -201,26 +189,12 @@ const mapDispatchToProps = dispatch => {
         return;
       }
 
-      const { element } = enabledElements[activeViewportIndex];
-
-      const eventData = {
-        event: {
-          clientX: event.clientX,
-          clientY: event.clientY,
-        },
-        element,
-      };
-
       const { toolType, measurementId } = measurementData;
       const tool = MeasurementApi.Instance.tools[toolType].find(measurement => {
         return measurement._id === measurementId;
       });
 
-      const options = {
-        editDescriptionOnDialog: true,
-      };
-
-      getMeasurementLocationCallback(eventData, tool, options);
+      ownProps.onEditDescription(tool);
```

Comment:
Is this required/guaranteed to be defined? Same with other callbacks.

Reply:
Alright, done. Thanks.